lib/thesis/view.ex

```diff
@@ -193,16 +193,21 @@ defmodule Thesis.View do
     raw """
     <script>
       ;(function () {
-        var loadThesis = function () {
+        var loadThesis = function (callback) {
```

Comment:
This script is injected with `thesis_editor(@conn)`. It loads Thesis asynchronously and then mounts Thesis.